docs/documenters/buildthedocs.rst

```diff
@@ -1,20 +1,59 @@
 Building the Docs Locally
 =========================
 
+This documentation is hosted on `readthedocs <https://docs.readthedocs.io/>`_,
+which is a documentation hosting site based on the
+`Sphinx <http://www.sphinx-doc.org/>`_ documentation tool,
+which in turn uses the `reStructuredText <http://docutils.sourceforge.net/rst.html>`_
```

Comment:
The official documentation for reStructuredText is a spectacularly ugly web page. Perhaps http://www.sphinx-doc.org/en/stable/rest.html might be a more useful link for someone who just wants a quick overview so that they can start making changes?